assets/app/data/crowbar/services/crowbar-backup.factory.js

```diff
@@ -23,9 +23,9 @@
 
             var requestOptions = {
                 method: 'POST',
-                url: '/utils/backups',
```

Comment:
The taken approach, with exception of backup, is to reflect the entities exposed by the API and let each controller decide which ones needs to be consumed during the upgrade flow.
The problem here are not the changes we need to do in the UI, but the uncompleted design of the API that force us as client to re-work our backend integration layer.